_posts/2020-02-14-numfocus-sponsored-project.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: MDAnalysis is a NumFOCUS Sponsored Project!
+---
+
+<img src="{{ site.baseurl }}{{ site.images }}/numfocus-sponsored.png"
+style="float: right" alt="NumFOCUS Sponsored" width="40%"/>
+
+**[MDAnalysis is now a NumFOCUS Sponsored Project]({{
+site.numfocus.sponsored_project }})!**
+
+Let us explain who MDAnalysis and NumFOCUS are and what being a
+sponsored project means:
+
+[**MDAnalysis**]({{ site.baseurl }}/about) is an open source software
+project that produces the *MDAnalysis Python library* for the analysis
+of computer simulations of many-body systems at the molecular scale,
+spanning use cases from interactions of drugs with proteins to novel
+materials. It is widely used in the scientific community and is
+written by scientists for scientists. The MDAnalysis Project is
+represented by the [MDAnalysis Core Developers]({{ site.baseurl
+}}/about#governance).
+
+[**NumFOCUS**](https://numfocus.org/) is an organization that promotes
+open practices in research, data, and scientific computing by serving
+as a fiscal sponsor for open source projects and organizing
+community-driven educational programs. In brief, they provide
+financial and logistic infrastructure to [many open source
+projects](https://numfocus.org/sponsored-projects) that fullfil
```

Comment:
`fulfil` of `fulfill`, I think, depending on country..

Reply:
changed